blog/webassembly_interaction_patterns.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: webassembly-patterns-command-reactor-library
+title: "WebAssembly Interaction Patterns: Command, Reactor, Library"
+--image: "/img/there-and-back-again.jpg"
+date: 2023-06-06T09:00:00+05:00
+author: "Kevin Hoffman"
+author_profile: "https://mastodon.world/@autodidaddict"
+categories: ["webassembly", "components", "patterns"]
+description: Exploring WebAssembly Interaction Patterns - Command, Reactor, and Library"
+draft: false
+---
+The first thing people need to learn when encountering WebAssembly for the first time is how to interact with the code inside the module.
+There are a number of patterns available for this, and in this blog post I'll cover the main three: _Command, Reactor_, and _Library_.
+
+<!-- truncate -->
+
+As we've mentioned in a number of posts and conference talks, WebAssembly's isolated sandbox forces people to use functions that
+can only accept and return numbers. _Bailey Hayes_ likes to refer to this aspect of WebAssembly as just _"[a bunch of numbers in a trenchcoat](https://youtu.be/6_BRLqxiZPU?t=459)"_. There are 
+plenty of libraries, wrappers, tools, and extensions that try and hide this basic fact, but it's important to understand underneath all of that, it's comes down to exchanging numbers.
```

Comment:
Add comma before underneath:

There are plenty of libraries, wrappers, tools and extensions that try and hide this basic fact, but it's important to understand, underneath it all, it comes down to exchanging numbers.